### ISSUE TITLE
Recombination coordinates 0-based?

### ISSUE THREAD

**Issue by cizydorczyk** [kwongj/maskrc-svg#7]
Hello,

First of all, I want to say great utility tool! Been thinking about writing something like this but this looks great. I am able to run it no problem, but wanted to ask one question regarding the output recombination coordinates.

The coordinate appear *almost* the same in the output file as in the input GFF, but they start 1 base earlier. I.e. if the GFF start is base 1000, it is reported in the output recombination file as 999, but the end coordinate is the same.

Would this be due to 0-based numbering of coordinates in python?

Thank you,
Conrad

**Comment by kwongj** [comment]
@cizydorczyk Thanks for picking this up!
This is very curious, because it only happens when parsing files from Gubbins, not ClonalFrameML. I think it has something to do with the Biopython GFF parser: `BCBio-GFF` module.
I can easily fix this in my script, but do you know if it starts masking 1 base too soon?

I have tested using CFML inputs previously which didn't have any issues.

**Comment by cizydorczyk** [comment]
@kwongj 
I'm not really sure what is going on, but I think there may be an issue with masking as well...

For example, with the following GFF entry:
`SEQUENCE	GUBBINS	CDS	3408918	3408930	0.000	.	0	node="internal_44->A058-E02-01-04-2013";neg_log_likelihood="41.409738";taxa="A058-E02-01-04-2013";snp_count="5";`

The reported recombinant positions are:
`A058-E02-01-04-2013	3408917	3408930	extant`

But when I actually examine the masked section in the sequences...
```
>>> print(seq[3408915:3408931])
GNNNNNNNNNNNNNNC
>>> print(seq2[3408915:3408931])
GTCATTATCACCCATC
```
Where `seq` is the masked sequence and `seq2` is the original.

So it it looks like it starts masking at index 3408916, which would correspond to (1-based) position 3408917...and goes up to index 3408929 (1-based position 34908930), which is what is reported in both the GFF and masked regions files. It masks 14 bases, although I would expect 13 from the range given 
in the GFF (i.e. 1 base too early).

Hope that helps!
Conrad

**Comment by kwongj** [comment]
@cizydorczyk Thanks - very helpful.
Have fixed this now in [v0.5](https://github.com/kwongj/maskrc-svg/releases/tag/v0.5) if you `git pull` in your directory.
Let me know if any further issues.
